src/community/graticule/src/main/java/org/geotools/data/graticule/GraticuleDataStore.java

```diff
@@ -0,0 +1,388 @@
+package org.geotools.data.graticule;
```

Comment:
Missing copyright header.
Also, while I have done this myself in the DGGS module, we should not have org.geotools classes in GeoServer.
Do you plan to move this store down to GeoTools eventually?

Reply:
Ideally, in the long run this could be useful in GeoTools but I'm not sure how easy it would be to move it.